cdc/model/sink.go

```diff
@@ -241,13 +242,127 @@ type RedoLog struct {
 	Type    RedoLogType          `msg:"type"`
 }
 
+// GetCommitTs returns the commit ts of the redo log.
+func (r *RedoLog) GetCommitTs() Ts {
+	switch r.Type {
+	case RedoLogTypeRow:
+		return r.RedoRow.Row.CommitTs
+	case RedoLogTypeDDL:
+		return r.RedoDDL.DDL.CommitTs
+	default:
+		log.Panic("invalid redo log type", zap.Any("type", r.Type))
+	}
+	return 0
+}
+
 // RedoRowChangedEvent represents the DML event used in RedoLog
 type RedoRowChangedEvent struct {
 	Row        *RowChangedEvent `msg:"row"`
 	PreColumns []*RedoColumn    `msg:"pre-columns"`
 	Columns    []*RedoColumn    `msg:"columns"`
 }
 
+// RedoDDLEvent represents DDL event used in redo log persistent
+type RedoDDLEvent struct {
+	DDL  *DDLEvent `msg:"ddl"`
+	Type byte      `msg:"type"`
+}
+
+// ToRedoLog converts row changed event to redo log
+func (row *RowChangedEvent) ToRedoLog() *RedoLog {
```

Comment:
Why moving the following conversion code from `cdc/redo/common` package to `cdc/model` package?

Reply:
This is because we cannot bind methods to a struct outside of the pkg.